### ISSUE TITLE
Build: some pages are built to wrong path

### ISSUE THREAD

**Issue by musicEnfanthen** [music-encoding/music-encoding.github.io#642]
There seems to be an inconsistency in the URL structure for certain pages (such as projects, tools, and tutorials). Specifically, these pages appear to be loosing a segment from their URLs when navigating to their sub pages.

For instance:

| Overview Page | Corresponding Sub Page |
|--------------|--------------------|
| https://music-encoding.org/community/projects-users.html | https://music-encoding.org/projects/aruspix.html |
| https://music-encoding.org/resources/tools.html | https://music-encoding.org/tools/encoding-tools |
| https://music-encoding.org/resources/tutorials.html | https://music-encoding.org/tutorials/100-structure.html |

In each of these examples, the corresponding page's URL is missing a segment from the overview page's URL. This could potentially lead to confusion.

For the conference pages this works fine:

| Overview Page | Corresponding Sub Page |
|--------------|--------------------|
| https://music-encoding.org/conference/past.html | https://music-encoding.org/conference/2022/ |

The issue is probably related to the setup of the projects/tools/tutorials collections in the _config.yml:

https://github.com/music-encoding/music-encoding.github.io/blob/71df1418a490c4cae2236dfe9b6a82978b3171d2/_config.yml#L111-L126

**Comment by bwbohl** [comment]
You're probably right and thanks for spotting, missed to check on that when updating the tools section…

**Comment by bwbohl** [comment]
So what do you think the URLs should look like?



Overview Page | Corresponding Sub Page
-- | --
https://music-encoding.org/community/projects-users.html | https://music-encoding.org/community/projects/aruspix.html
https://music-encoding.org/resources/tools.html | https://music-encoding.org/resources/tools/encoding-tools
https://music-encoding.org/resources/tutorials.html | https://music-encoding.org/resources/tutorials/100-structure.html

**Comment by musicEnfanthen** [comment]
Yes, exactly, that would keep the structure of the subpages intact.
